core/testdata/transcript_go_proto/transcript.proto

```diff
@@ -34,10 +34,9 @@ message Transcript {
 // Action contains one request and one response.
 message Action {
    string desc = 1;
-   // trust_new_log controls the client verifier state.
-   // If true, the client records the resp.LatestLogRoot and uses it
-   // for the next rpc's request.last_verified_tree_size.
-   bool trust_new_log = 2;
+   reserved 2;  // was trust_new_log
+   // last_verified_log_root encodes the client verifier state.
```

Comment:
optional: In Go generated code the field name is formatted as `LastVerifiedLogRoot`, which is not consistent with formatting in the comment. My strategy is to not start comments with field names in proto files.

Reply:
Alas, this is a common problem. I used to use the Go capitalization so the comments would come out correctly, but the mixing of different capitalization schemes in the proto file seemed messy and language specific. 

Unsure if not starting comments with field names fixes the problem, since doing so still violates go style in the generated code.